core/bcast/bcast.go

```diff
@@ -63,13 +65,27 @@ func (b Broadcaster) Broadcast(ctx context.Context, duty core.Duty,
 			return err
 		}
 
-		return b.eth2Cl.SubmitAttestations(ctx, []*eth2p0.Attestation{att})
+		err = b.eth2Cl.SubmitAttestations(ctx, []*eth2p0.Attestation{att})
+		if err == nil {
```

Comment:
use either `err != nil` first or `err == nil` consistently, though i'd prefer the first one

Reply:
done